app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -0,0 +1,94 @@
+package org.fossasia.openevent.data;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Created by MananWason on 27-05-2015.
+ */
+public class Event {
+    int id;
+    String name;
+    String logo;
+    SimpleDateFormat start;
+    SimpleDateFormat end;
+    float latitude;
+    float longitude;
+    String location_name;
```

Comment:
http://en.wikipedia.org/wiki/Hungarian_notation this. Your var name can be in hungarian notation, like `locationName` but for Serialization purposes like Gson converting from JSON to Java bean it will use `location_name` as the json key, this is for mapping purposes.